src/vm/methodtablebuilder.cpp

```diff
@@ -1548,8 +1546,10 @@ MethodTableBuilder::BuildMethodTableThrowing(
         MethodTable* pMTParent = GetParentMethodTable();
```

Comment:
I do not think we need to bother will all COM interop specific validations without `FEATURE_COMINTEROP`. It will be incomplete anyway.

Just set `fIsComObjectType` bit and be done with it.

Reply:
Okay. working on